build/configs/default_config.yaml

```diff
@@ -150,6 +150,7 @@ ethereum:
     anchorCommit: 85000
     anchorPreCommit: 100000
     nftMint: 900000
+    nftTransferFrom: 900000
```

Comment:
:O

Reply:
Need to replace it with the correct amount + buffer :)